govgen-1/README.md

```diff
@@ -0,0 +1,36 @@
+# 🔗 `govgen-1`
+
+![chain-id](https://img.shields.io/badge/chain%20id-govgen--1-blue?style=for-the-badge)
+
+## Register in the Genesis
+
+To register your validator node in the `genesis.json` you just need to provide a signed `gentx` with an initial delegation of `10000000000ugovgen`.
+
+```sh
+# Init node
+govgend init your-moniker --chain-id govgen-1
+
+# Create keys, be careful with the mnemonic 👀
+govgend keys add your-key-name
+
+# Set account necessary balance
+govgend genesis add-genesis-account your-key-name 10000200000ugovgen
```

Comment:
1. What's the purpose of the extra `200000`, for gas?
2. Is it mandatory to set `10000200000` for balance or can it be bigger ?

Reply:
You raise a good point.
Let me try to clarify, and at the same time let me say in advance this number will likely change soon.

In general, since tokens aren't transferrable on GovGen any account that did not get a token allocation - meaning did not vote NO/NWV on prop 848 on the Cosmos Hub - won't be able to do anything on the chain let alone become a validator.

Post launch, any account with tokens will be able to spin up a validator if they want, permissionlessly.

But before launch, we also want to give the possibility for people interestend to join the valset even if they do not have balance on GovGen - or for security reasons want to use another account.
In this case though, we will create these genesis account but will grant them **only the absolute mininum necessary** for operations, so they won't have any relevant voting power.